trl/trainer/gkd_trainer.py

```diff
@@ -179,17 +179,21 @@ def generalized_jsd_loss(
         student_logits = student_logits / temperature
         teacher_logits = teacher_logits / temperature
 
+        # Compute the interpolated probabilities
+        student_probs = F.softmax(student_logits, dim=-1)
+        teacher_probs = F.softmax(teacher_logits, dim=-1)
+        mixture_probs = beta * student_probs + (1 - beta) * teacher_probs
+        # Compute the log of the mixture distribution
+        mixture_log_prob = mixture_probs.log()
```

Comment:
probs are around 1e-20 on average, don't you think we can have numerical issues here?

Maybe keeping log-probs is safer:

```python
# log(a + b) = log(exp(log(a)) + exp(log(b))) -> for mixture
mixture_log_probs = torch.logsumexp(torch.stack([
    log_student_probs + torch.log(torch.tensor(beta)),
    log_teacher_probs + torch.log(torch.tensor(1 - beta))
]), dim=0)
```